.github/workflows/integration.yml

```diff
@@ -51,3 +51,7 @@ jobs:
       - name: Remove test logs
         if: always()
         run: rm -rf /tmp/goth-tests
+
+      # Only relevant for self-hosted runners
+      - name: Remove poetry virtual env
```

Comment:
Let's add `if: always()` here to always attempt removing the env, regardless of previous steps failing.